tests/testdata/provider/testdata_oracle.sql

```diff
@@ -65,5 +65,16 @@ CREATE TABLE QGIS.DATE_TIMES ( "id" INTEGER PRIMARY KEY, "date_field" DATE, "dat
 
 INSERT INTO QGIS.DATE_TIMES ("id", "date_field", "datetime_field" ) VALUES (1, DATE '2004-03-04', TIMESTAMP '2004-03-04 13:41:52' );
 
+CREATE TABLE QGIS.POINT_DATA_IDENTITY ( "pk" NUMBER GENERATED ALWAYS AS IDENTITY PRIMARY KEY, GEOM SDO_GEOMETRY);
+INSERT INTO QGIS.POINT_DATA_IDENTITY (GEOM)
+ SELECT SDO_GEOMETRY( 2001,4326,SDO_POINT_TYPE(1, 2, NULL), NULL, NULL) from dual
```

Comment:
Is there a specific reason to add so much elements. It's no big deal anyway, I'm just wondering

Reply:
No, it's true, one is enough. I changed it !